pvlib/tracking.py

```diff
@@ -190,7 +198,17 @@ def singleaxis(apparent_zenith, apparent_azimuth,
 
     # NOTE: max_angle defined relative to zero-point rotation, not the
     # system-plane normal
-    tracker_theta = np.clip(tracker_theta, -max_angle, max_angle)
+
+
+    # Determine minimum and maximum rotation angles for the tracker based on max_angle.
+    # If max_angle is a single value, assume min_angle is the negative of max_angle.
+    if np.array(max_angle).size == 1:
```

Comment:
```suggestion
    if np.isscalar(max_angle):
```
Maybe a bit easier to read while achieving the same thing?